pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerStarter.java

```diff
@@ -193,6 +194,29 @@ public Long call() throws Exception {
       }
     });
 
+    ServiceStatus.setServiceStatusCallback(new ServiceStatus.ServiceStatusCallback() {
```

Comment:
Doesn't really need to, since it won't get reused anywhere else anyway. This behavior is controller-specific.